Chart/Chart.swift

```diff
@@ -451,6 +463,38 @@ class Chart: UIControl {
         return lineLayer
     }
     
+    private func drawPoints(xValues xValues: Array<Float>, yValues: Array<Float>, seriesIndex: Int, i: Int) -> CAShapeLayer {
+        
+        let circleLayer = CAShapeLayer()
+        let circleRadius: CGFloat = 2.0
+        
+        func circleFrame(x: CGFloat, y: CGFloat) -> CGRect {
+            var circleFrame = CGRect(x: x, y: y, width: 2*circleRadius, height: 2*circleRadius)
+            circleFrame.origin.x = x - circleRadius
+            circleFrame.origin.y = y - circleRadius
+            return circleFrame
+        }
+        
+        func circlePath(i: Int) -> UIBezierPath {
+            let x = xValues[i]
+            let y = yValues[i]
+            return UIBezierPath(ovalInRect: circleFrame(CGFloat(x), y: CGFloat(y)))
+        }
+        
+        circleLayer.frame = self.bounds
+        circleLayer.path = circlePath(i).CGPath
+        
+        circleLayer.lineWidth = pointWidth
+        circleLayer.fillColor = series[seriesIndex].colors.above.CGColor
+        circleLayer.strokeColor = series[seriesIndex].colors.above.CGColor
```

Comment:
Each chart series [is using](https://github.com/gpbl/SwiftChart/blob/master/Chart/ChartSeries.swift#L22) a `(above, below)` tuple ([example](https://cloud.githubusercontent.com/assets/120693/11602672/5c303ac6-9adc-11e5-9006-3275a16b7ec8.png)), so I'm not sure – why did you choose to use the `above` color?